controller/jobs_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"strings"
 
 	. "github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-check"
+	controller "github.com/flynn/flynn/controller/client"
```

Comment:
I think explicitly identifying a package whose name is different from its path is a good idea.